<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HoneyPipe</string>
  </property>
  <property name="windowOpacity">
   <double>0.920000000000000</double>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="cbInt">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblInt">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>121</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose interface</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIp">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>55</y>
      <width>70</width>
      <height>10</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose IP</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbIp">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>45</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnScan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Scan</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twMain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>137</y>
      <width>452</width>
      <height>331</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>100</number>
    </attribute>
    <column>
     <property name="text">
      <string>IP</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MAC</string>
     </property>
    </column>
   </widget>
   <widget class="QLineEdit" name="leSource">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>532</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leDest">
    <property name="geometry">
     <rect>
      <x>334</x>
      <y>532</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>535</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;     Me (MITM)     &gt;&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>509</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Source Victim :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>510</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Dest Victim (usually router):</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSetSource">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>477</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSetDest">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>477</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbSpoof">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Start spoofing</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPlay">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbProtocol">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Netsoul</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>509</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="sbMain">
   <property name="toolTip">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
